mac

```diff
@@ -200,7 +202,6 @@ if [ ! $SKIP_HOMEBREW_UPDATE ]; then
     brew "openssl@1.1"
     brew "the_silver_searcher"
     brew "vim"
-    brew "zsh"
```

Comment:
[zsh is the default Mac shell](https://support.apple.com/en-ca/guide/terminal/trml113/mac#:~:text=The%20default%20shell%20is%20zsh,windows%20and%20tabs%20open%20with.&text=In%20the%20Terminal%20app%20on,shell%20you%20want%20to%20use.), we no longer need to install it as a brew formulae